### ISSUE TITLE
Shopify App generator problem

### ISSUE THREAD

**Issue by resistorsoftware** [Shopify/shopify_app#591]
Rails 5.2, shopify_app gem 8.2.6 and new fresh rails install

run default generator

    rails generate shopify_app:install --api_key ff00ff00 --secret ccddee1199

Blows chunks with a:

    /Users/dlazar/.rvm/gems/ruby-2.4.2/gems/railties-5.2.0/lib/rails/railtie/configuration.rb:97:in 
    `method_missing': undefined method `assets' for # 
   <Rails::Application::Configuration:0x00007f89e4cba4b0>
    Did you mean?  asset_host (NoMethodError)

    module ShopifyApp
      class Engine < Rails::Engine
        engine_name 'shopify_app'
        isolate_namespace ShopifyApp

        initializer "shopify_app.assets.precompile" do |app|
         app.config.assets.precompile += %w( shopify_app/redirect.js )
        end
      end
    end

So I bundle open shopify_app and see the assets method on the app.config object. What is up with this?

**Comment by vfonic** [comment]
I haven't had this issue. You could see that in a video I posted in another issue. I successfully setup Shopify app using rails 5.2 and shopify_app 8.2.6

**Comment by resistorsoftware** [comment]
I was only trying Rails to quickly mimic the Webpack setup to try and steal some of that awful config info for my Sinatra App. I am stepping away from using Rails, quietly, to let it steep for another few years... too many problems are Rails, not enough solution.

**Comment by vfonic** [comment]
Then you hit a real bug in shopify_app! It's dependent on sprockets being available.

You should reopen this issue and rename it to something that reflects that. That would really help maintainers to issue the next version with this fixed!

**Comment by resistorsoftware** [comment]
True. I had no use for Sprockets, so I took the two seconds needed to re-spawn the same product with Sprockets, and then tried the exact same generator, and the whole thing HUNG. Dead. So at that point I quickly realized I was working with out-to-lunch code. I fell back to older Rails, 5.1.5 that I thought might work, it failed in the same way. I realized I was only getting success because I always skip using generators and hand-code that stuff... Hence my lack of enthusiasm for Rails.
